odc/stats/plugins/l34_utils/l4_natural_aquatic.py

```diff
@@ -5,400 +5,282 @@
+    res = expr_eval(
+        "where((a==56)&(b==2), 253, a)",
         {
-            "a": l4,
-            "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
+            "a": res,
+            "b": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_water_season",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==13)&(c==1)&(d==2), 71, a)",
+    res = expr_eval(
+        "where((a==57)&(b==1), 252, a)",
         {
-            "a": l4,
-            "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
+            "a": res,
+            "b": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_water_season",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==15)&(c==1)&(d==1), 73, a)",
+    res = expr_eval(
+        "where((a==57)&(b==2), 251, a)",
         {
-            "a": l4,
-            "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
+            "a": res,
+            "b": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_water_season",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==15)&(c==1)&(d==2), 74, a)",
+    # mark final
+
+    res = expr_eval(
+        "where((a==254)&(b==10), 64, a)",
         {
-            "a": l4,
+            "a": res,
             "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==16)&(c==1)&(d==1), 76, a)",
+    res = expr_eval(
+        "where((a==253)&(b==10), 65, a)",
         {
-            "a": l4,
+            "a": res,
             "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==16)&(c==1)&(d==2), 77, a)",
+    res = expr_eval(
+        "where((a==252)&(b==10), 80, a)",
         {
-            "a": l4,
+            "a": res,
             "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==10)&(c==2)&(d==1), 79, a)",
+    res = expr_eval(
+        "where((a==251)&(b==10), 81, a)",
         {
-            "a": l4,
+            "a": res,
             "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==10)&(c==2)&(d==2), 80, a)",
+    #########################################
```

Comment:
unwanted artefact

Reply:
it's not unwanted.  It's meant to be there to section the code

> unwanted artefact